src/docs/spec.md

```diff
@@ -370,11 +372,13 @@ We recommend to use the following *filename conventions* for SSSOM metadatafiles
 Example ([download](https://raw.githubusercontent.com/mapping-commons/sssom/master/examples/external/mp-hp-exact-0.0.1.sssom.yml)):
 
 ```
-creator_id: "https://orcid.org/0000-0002-7356-1779"
 curie_map: 
   HP: "http://purl.obolibrary.org/obo/HP_"
   MP: "http://purl.obolibrary.org/obo/MP_"
-  skos: "http://www.w3.org/2004/02/skos/core"
+  skos: "http://www.w3.org/2004/02/skos/core#"
```

Comment:
```suggestion
  semapv: "https://w3id.org/semapv/vocab/"
  skos: "http://www.w3.org/2004/02/skos/core#"
```
I suggest adding `semapv` to the curie map so that all the prefixes used in the example are included.

Reply:
No. `semapv` is a built-in prefix, it _may_ be listed in the prefix map but that is not required. `skos` is not required either in fact, but it was in the original example so I left it.

Reply:
Thanks for removing `skos` from the examples. Since `skos` and `semapv` are both built-in prefixes, I felt they should either both be included or both be excluded from the curie map.

Reply:
Yes, I agree with that. This also has the side-effect of fixing the position of the incorrectly sorted `orcid:` prefix, since the `skos:` prefix is no longer in between. :)